src/main/java/io/anserini/search/SearchCollection.java

```diff
@@ -369,10 +485,9 @@ public static void main(String[] args) throws Exception {
 
     final long start = System.nanoTime();
     SearchCollection searcher = new SearchCollection(searchArgs);
-    int numTopics = searcher.runTopics();
+    searcher.runTopics();
     searcher.close();
     final long durationMillis = TimeUnit.MILLISECONDS.convert(System.nanoTime() - start, TimeUnit.NANOSECONDS);
-    LOG.info("Total " + numTopics + " topics searched in "
-        + DurationFormatUtils.formatDuration(durationMillis, "HH:mm:ss"));
+    LOG.info("Total run time: " + DurationFormatUtils.formatDuration(durationMillis, "HH:mm:ss"));
```

Comment:
Why don't we print out number of topics anymore?

Reply:
Because now it is print inside each retrieval thread. 
Search for `LOG.info("Run " + topics.size()` in this PR